src/views/WorkflowTable.vue

```diff
@@ -103,17 +106,18 @@ subscription Workflow {
 fragment WorkflowData on Workflow {
   id
   status
+  cylcVersion
   owner
   host
   port
 }
 `
 
 export default {
-  name: 'WorkflowsTable',
+  name: 'WorkflowTable',
 
   mixins: [
-    subscriptionMixin
```

Comment:
This seems to have been the wrong mixin. The workflows view was only working because the data it needed happened to have been requested (in error) by the workflows view.